### PR TITLE
Debugger: show button shortcuts in tooltips

### PR DESCRIPTION
**Pull request by jess-x**
## References

Related to https://github.com/jupyterlab/jupyterlab/issues/10139

>10. Make the hotkeys for the debugger more obvious by showing them in the tooltip on the various button

## User-facing changes

Before:
Screen Recording 2021-05-05 at 6 55 10 PM

Now: 
Screen Recording 2021-05-05 at 6 58 23 PM

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks @jess-x for starting this :+1:

File: packages/debugger-extension/src/index.ts [hunk 1]
```diff
@@ -505,7 +507,7 @@ const main: JupyterFrontEndPlugin<void> = {
 
     commands.addCommand(CommandIDs.debugContinue, {
       label: trans.__('Continue'),
-      caption: trans.__('Continue'),
+      caption: trans.__('Continue (F9)'),
```
Comment (jtpio) [inline packages/debugger-extension/src/index.ts:510]: As an alternative to encoding the keybinding in the caption string, maybe if would also be possible to get the keybinding with the `CommandRegistry.formatKeystroke` method?

The approach would be similar to what is currently used for the simple mode switch in the status bar:

https://github.com/jupyterlab/jupyterlab/blob/d065f9442332f90128a7436561d0f99b6a591107/packages/statusbar-extension/src/index.ts#L370-L381

Comment (goanpeca) [reply packages/debugger-extension/src/index.ts:510]: Agreed! thanks for the tip @jtpio :)

File: packages/apputils/src/toolbar.tsx [hunk 2]
```diff
@@ -670,7 +670,13 @@ namespace Private {
     if (!commands.isVisible(id, args)) {
       className += ' lm-mod-hidden';
     }
-    const tooltip = commands.caption(id, args) || label || iconLabel;
+    let tooltip = commands.caption(id, args) || label || iconLabel;
+    // Shows hot keys in tooltips
+    const binding = commands.keyBindings.find(b => b.command === id);
+    if (binding) {
+      const ks = CommandRegistry.formatKeystroke(binding.keys.join(' '));
+      tooltip = tooltip + ' (' + ks + ')';
```
Comment (goanpeca) [inline packages/apputils/src/toolbar.tsx:678]: How about

```typescript
tooltip = `${tooltip} (${ks})`;
```

**Review by goanpeca — APPROVED** [review]
Works as expected and tooltips are correctly updating the shortcut when change in the settings.

Thanks @jtpio for the suggestions.

Thanks for working on this @jess-x 🚀

**Comment by jess-x** [comment]
Thank you @jtpio for the tip, indeed! 
I guess sometimes just need to restart the checks :) @goanpeca 
I am glad to help. I've definitely learned more about React working on this

**Review by jtpio — APPROVED** [review]
Thanks!

**Comment by meeseeksmachine** [comment]
This pull request has been mentioned on **Jupyter Community Forum**. There might be relevant details there:

https://discourse.jupyter.org/t/shortcuts-for-stepping-through-break-points-in-debugger/9559/2

**Pull request merged**